pkg/systemd/init.js

```diff
@@ -218,6 +230,24 @@ define([
                 });
         }
 
+        function add_timer_properties(timer_unit,unit) {
+            unit.LastTriggerTime = moment(timer_unit.LastTriggerUSec/1000).calendar();
+            if (timer_unit.LastTriggerUSec === 0)
+                unit.LastTriggerTime = _("not known");
+            var next_run_time = 0;
+            if (timer_unit.NextElapseUSecRealtime === 0)
+                next_run_time = timer_unit.NextElapseUSecMonotonic + systemd_manager.GeneratorsStartTimestamp;
+            else if (timer_unit.NextElapseUSecMonotonic === 0)
+                next_run_time = timer_unit.NextElapseUSecRealtime;
+            else {
+                if (timer_unit.NextElapseUSecMonotonic + systemd_manager.GeneratorsStartTimestamp < timer_unit.NextElapseUSecRealtime)
+                    next_run_time = timer_unit.NextElapseUSecMonotonic + systemd_manager.GeneratorsStartTimestamp;
+                else
+                    next_run_time = timer_unit.NextElapseUSecRealtime;
+            }
+            unit.NextRunTime = moment(next_run_time/1000).calendar();
+        }
```

Comment:
I found in [init.js](https://github.com/cockpit-project/cockpit/blob/master/pkg/systemd/init.js) that variables are written like next_run_time, active_state, systemd_manager etc while properties of unit are like unit.LoadState, unit.UnitFileState, unit.CombinedState et cetera.
so I think having properties as unit.NextRunTime, unit.LastTriggerTime and variables as next_run_time is better than changing?